gazel.go

```diff
@@ -162,6 +162,7 @@ func (v *Venderor) updatePkg(path, _ string, pkg *build.Package) error {
 
 	var attrs Attrs = make(Attrs)
 	srcs := asExpr(merge(pkg.GoFiles, pkg.SFiles)).(*bzl.ListExpr)
+	cgoSrcs := asExpr(merge(pkg.CgoFiles, pkg.CFiles, pkg.CXXFiles, pkg.HFiles)).(*bzl.ListExpr)
```

Comment:
For whatever reason, I think this is how they implemented it. https://github.com/bazelbuild/rules_go#cgo_library